apps/ledger-live-desktop/src/newArch/features/ModularDrawer/components/SelectAccountFlow.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import SelectAccountAndCurrencyDrawer from "~/renderer/drawers/DataSelector/SelectAccountAndCurrencyDrawer";
+
+import { CryptoOrTokenCurrency } from "@ledgerhq/types-cryptoassets";
+import { AccountLike, Account } from "@ledgerhq/types-live";
+import { WalletAPIAccount } from "@ledgerhq/live-common/wallet-api/types";
+import { Observable } from "rxjs";
+
+type SelectAccountDrawerProps = {
+  assetIds?: string[];
```

Comment:
I'm not sure what will be the currencies prop ultimately (once we actually implement the asset/network separation) but I think that you are right in the meantime we can just use `currencies: CryptoOrTokenCurrency[]` however in that case lets remove `assetIds` and `includeTokens` since they are useless when currencies is mandatory.